app/lib/modules/llm/manager.ts

```diff
@@ -187,18 +211,21 @@ export class LLMManager {
 
     return modelList;
   }
+
   getStaticModelListFromProvider(providerArg: BaseProvider) {
     const provider = this._providers.get(providerArg.name);
 
     if (!provider) {
       throw new Error(`Provider ${providerArg.name} not found`);
     }
 
-    return [...(provider.staticModels || [])];
+    // 화이트리스트 적용
+    return filterWhitelistedModels(provider.staticModels || []);
   }
 
   getDefaultProvider(): BaseProvider {
-    const firstProvider = Array.from(this._providers.values()).find((p) => p.name === 'OpenRouter');
+    // 화이트리스트에 있는 첫 번째 프로바이더를 기본값으로 사용
+    const firstProvider = Array.from(this._providers.values())[0];
```

Comment:
it seems that `[0]` indexing will change behavior(i.e., checking `if (!firstProvider)`) when `this._providers.values()` is empty.